shoop/core/modules/interface.py

```diff
@@ -31,10 +31,10 @@ def _load_module(self):
         if module_identifier:
             impls = self.get_module_implementation_map()
             if module_identifier not in impls:
-                raise ModuleNotFound("Invalid module identifier %r in %r" % (module_identifier, self))
+                raise ModuleNotFound("Invalid module identifier %r in %r" % (module_identifier, force_ascii(self)))
```

Comment:
This should now be

```python
raise ModuleNotFound("Invalid module identifier %r in %s" % (module_identifier, force_ascii(repr(self))))
```

to retain the old behavior. (Otherwise this would be `force_ascii()`ing the `str()` representation of the object, not the `repr`).